docs/whats_new/v0.1.rst

```diff
@@ -25,6 +25,7 @@ Version 0.1
 
 Changelog
 ---------
+- |Feature| Introduce chain graphs and validity checks, and refactor CPDAG and chain graphs to use a directed-undirected private class, by `Jaron Lee`_ (:pr:`73`)
```

Comment:
Typically change log goes on the bottom. I think the Changelog CI check should get fixed that way

Reply:
oh fair enough I was just following https://keepachangelog.com/en/1.0.0/ which has everything in reverse chronological order